_posts/2018-03-26-iviewer-1.0.0.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Release of OMERO.iviewer 1.0.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.iviewer 1.0.0
+---
+Today we are releasing version 1.0.0 of OMERO.iviewer: a full production-ready
+OMERO.web app for visualizing, annotating and analyzing images in OMERO. This
```

Comment:
'annotating' might be confused with adding OMERO annotations (not yet supported) rather than drawing ROIs, but if this is considered close enough then I'm OK with it.

Reply:
I think @pwalczysko suggested it - I would say that drawing and *labelling* ROIs count as annotating but happy to change it if people think it'll be confusing